website/docusaurus.config.js

```diff
@@ -519,11 +512,18 @@ const config = {
     }),
 };
 
-// TODO temporary dogfood async config, remove soon
 async function createConfig() {
-  await new Promise((resolve) => {
-    setTimeout(resolve, 0);
-  });
+  const FeatureRequestsPlugin = (await import('./src/featureRequests/FeatureRequestsPlugin.mjs')).default;
```

Comment:
Yea, but that creates too much indentation changes... I'd rather use this with the return of less indentation for our massive `config` object

Reply:
> I'd rather use this with the return of less indentation for our massive config object

I don't understand what you mean here

Do you see this as a temporary step?

Reply:
No, I view this as a long term solution.

```js
const config = {
  foo: bar, // two spaces of indentation
};
```

```js
async function createConfig() {
  return {
    foo: bar, // four spaces of indentation
  };
}
```

I'd like to avoid that extra level of indentation at all costs

Reply:
I'd 1000% prefer to have an extra indentation than to have this in the long term (which we currently have):

```js
const config = {
  foo: bar, // two spaces of indentation
};

async function createConfig() 
  config.xyz = await abc();
  return config;
}

module.exports = createConfig;
```

Reply:
Ha, I'm not sure if we are to add more async items in the future. We'll see...

Reply:
We can keep it for now, I'll figure out a way to clean this up later